NextOfKin/Classes/KinSdkController.swift

```diff
@@ -128,8 +94,8 @@ public class KinSdkController: KinControllerType, KinRespositoryType {
                 return Disposables.create()
             }
             do {
-                this.kinClient = try KinClient(provider: this.getServiceProvider())
```

Comment:
so we're never setting and holding a ref to kinClient? I actually think there is a bug somewhere if we don't hold a ref to it...remember seeing this during hackweek...something breaks inside KinSDK, they're expecting you to keep a ref to the client =/